fabric/src/main/java/com/noxcrew/noxesium/network/clientbound/ClientboundCustomSoundVolumePacket.java

```diff
@@ -0,0 +1,40 @@
+package com.noxcrew.noxesium.network.clientbound;
+
+import com.noxcrew.noxesium.feature.sounds.NoxesiumSoundInstance;
+import com.noxcrew.noxesium.feature.sounds.NoxesiumSoundModule;
+import com.noxcrew.noxesium.network.NoxesiumPackets;
+import net.fabricmc.fabric.api.networking.v1.PacketSender;
+import net.fabricmc.fabric.api.networking.v1.PacketType;
+import net.minecraft.client.player.LocalPlayer;
+import net.minecraft.network.FriendlyByteBuf;
+import net.minecraft.resources.ResourceLocation;
+
+/**
+ * Sent by a server to change the volume of a sound. The interpolation time can be
+ * used to fade the sound up or down over an amount of ticks
+ */
+public class ClientboundCustomSoundVolumePacket extends ClientboundNoxesiumPacket {
+    ResourceLocation soundLocation;
```

Comment:
What I worry about here is how to identify a specific sound. Could the custom play sound packet include some id int that you can reference back to here? Feels better than to allow any sound to be picked.

Reply:
That's fair,
I opted to use the sounds location because the original way I planned on using this was for music, and this was the simplest way to do so.
I do agree though, an id would work a lot better